### PR TITLE
Adding concepts docs, structure for CDK documentation

### PR DESCRIPTION
**Pull request by avaidyanatha**
## Main Changes
- Adding skeleton structure for our Python CDK documentation
- Adding concepts documentation to the docs website

## Misc Changes
- Updating the pull request template for a new connector to include a link to the UUID generator

## Notes
- Instead of copying the documentation to the `/docs` repo which could lead to desync, I suggest that we should keep and maintain all of our documentation in `/airbyte_cdk/python/docs`, which is what I did in this PR.

### CODE REVIEW

**Comment by michel-tricot** [comment]
what does it mean for our gitbook docs? Will it be available there?

If we are afraid about de-sync given our current setup, I would prefer that it lives with the gitbook docs than in the cdk package.

**Comment by avaidyanatha** [comment]
> what does it mean for our gitbook docs? Will it be available there?
> 
> If we are afraid about de-sync given our current setup, I would prefer that it lives with the gitbook docs than in the cdk package.

Yep, it will be available on our Gitbook docs even if it doesn't live in the `/docs` folder. For example, our `Introduction` section doesn't live in our `/docs` directory and is still available on the Gitbook site.

File: docs/SUMMARY.md [hunk 1]
```diff
@@ -91,21 +91,27 @@
   * [Code of Conduct](contributing-to-airbyte/code-of-conduct.md)
   * [Developing Locally](contributing-to-airbyte/developing-locally.md)
   * [Developing Connectors](contributing-to-airbyte/building-new-connector/README.md)
+    * [Using the Airbyte CDK](contributing-to-airbyte/using-the-airbyte-cdk/README.md)
```
Comment (sherifnada) [inline docs/SUMMARY.md:94]: For this README.md why don't we add something like 

```
Airbyte Connector Development Kits (CDK) which significantly accelerate connector development. We currently provide a Python CDK, and a Java one is in development. 

Navigate here for more information on the [Python CDK](link)
```

**Review by sherifnada — APPROVED** [review]
nice find!

File: docs/SUMMARY.md (hunk 1 above)
Comment (avaidyanatha) [reply docs/SUMMARY.md:94]: Good idea, done.

**Pull request merged**